+++
title = "CWE-584: Return Inside Finally Block\n"
description = "The code has a return statement inside a finally block, which will cause any thrown exception in the try block to be discarded.\n"
weight = 584

[extra]
id = 584
name = "Return Inside Finally Block\n"
url = "https://cwe.mitre.org/data/definitions/584.html"
class = "Base"
rust_docs_links = []
parent = "705"

[extra.vote]
"No Help, or Langs Wont Help" = false
Discouraged = false
"Discouraged via Library" = false
"Discouraged via Borrow Checker" = false
"Discouraged via Debug Mode" = false
"Discouraged via Clippy" = false
"Virtually Impossible" = true

[extra.vector]

+++

Finally isn't in std, though the abstraction is in common libraries